### ISSUE TITLE
Can not send event to channel after redeploy

### ISSUE THREAD

**Issue by SergeyMagid** [vert-x3/vertx-jgroups#10]
Hi All,

I have found a bug with this clustering implementation. 
JUnit test for this bug: https://github.com/SergeyMagid/vertx-jgroups/commit/2f1c8f97b7bf49634d84fe05234db2adbfcc3871

Steps to reproduce.
1. I have two nodes.
2. First node simple ping-pong application.
3. Second node send ping to first node but not directly. They have two verticles inside. One node periodically call other to make ping to first node.
i.e.

```
vertices[1].eventBus().consumer(WRAPPED_CHANNEL, wrapperEvent -> {
            EventBus ebSender = vertices[1].eventBus();
            ebSender.send(PING, "foo" + (int) wrapperEvent.body(), reply -> {
                assertEquals("PONG", reply.result().body().toString());
            });
        });
```
1. redeploy first node.
2. try send event to first node. (send event to WRAPPED_CHANNEL)

Result:
WRAPPED_CHANNEL get HANDLER_TIMEOUT.
But if send event to PING all work fine without WRAPPED_HANDLER.

I can provide two simple applications for reproduce this bug if this is not clear.

**Comment by purplefox** [comment]
Hi @fmarinelli any feedback on this? :)

**Comment by purplefox** [comment]
Any feedback Fabio? 

As per the roadmap, the Vert.x 3.2 release is due https://github.com/vert-x3/wiki/wiki/Vert.x-Roadmap

If we don't know if the JGroups cluster manager is ready or not it will miss the release.

**Comment by fmarinelli** [comment]
I'm working on it. Yes I know the roadmap and I will do anything to stay on track.

**Comment by fmarinelli** [comment]
Fixed with the PR #14
